_pages/04-data.md

```diff
@@ -115,3 +115,19 @@ The M-Lab *test name* Data Set, *date range used*. *M-Lab test URL*
 For example:
 
 The M-Lab NDT Data Set 2009-02-11–2015-12-21. [https://measurementlab.net/tests/ndt]({{ site.baseurl }}/tests/ndt)
+
+or, in [BibTeX](https://en.wikipedia.org/wiki/BibTeX) format:
```

Comment:
Last one :) - can we add the new window target code to this link: `[BibTeX](https://en.wikipedia.org/wiki/BibTeX){:target="_blank"}`

Reply:
And one more correction identified by the linter:
`_pages/tests/ndt/ndt.md:55: MD012 Multiple consecutive blank lines`